src/taipy/logger/_taipy_logger.py

```diff
@@ -35,7 +35,7 @@ def _get_logger(cls):
             cls.__logger.setLevel(logging.INFO)
             ch = logging.StreamHandler(sys.stdout)
             ch.setLevel(logging.INFO)
-            formatter = logging.Formatter("[%(asctime)s][%(name)s][%(levelname)s] %(message)s")
+            formatter = logging.Formatter("[%(asctime)s][%(name)s][%(levelname)s] %(message)s", "%Y-%m-%d %H:%M:%S")
```

Comment:
This is just to improve the time format of the logger.

Previously, the timestamp was like `2023-03-06 14:24:07,414` which includes the micro second.
Now it will only be `2023-03-06 14:24:07`.